pkg/log/log.go

```diff
@@ -17,6 +17,7 @@ type Logger interface {
 	Warnf(format string, args ...interface{})
 	Error(args ...interface{})
 	Errorf(format string, args ...interface{})
+	Panic(args ...interface{})
```

Comment:
Please, don't add such methods if not really necessary. see my comment in `metrics.go` file